test/token/ERC7984/extensions/ERC7984Rwa.test.ts

```diff
@@ -451,7 +451,7 @@ describe('ERC7984Rwa', function () {
         await token.connect(agent1).getHandleAllowance(frozenHandle, agent1, true);
         await expect(
           fhevm.userDecryptEuint(FhevmType.euint64, frozenHandle, await token.getAddress(), agent1),
-        ).to.eventually.equal(50); // frozen is left unchanged
+        ).to.eventually.equal(25); // frozen is left unchanged
```

Comment:
Mixing frozen & unfrozen, here is an example of inconsistency:
https://github.com/OpenZeppelin/openzeppelin-confidential-contracts/blob/7d0a1aaf607944f35bc76be0ea5d68dd3ba923de/test/token/ERC7984/extensions/ERC7984Rwa.test.ts#L404-L406
and:
https://github.com/OpenZeppelin/openzeppelin-confidential-contracts/blob/7d0a1aaf607944f35bc76be0ea5d68dd3ba923de/test/token/ERC7984/extensions/ERC7984Rwa.test.ts#L454